cert/cert.go

```diff
@@ -292,3 +292,18 @@ func ParseCertAndKey(certPEM, keyPEM string) (*x509.Certificate, *rsa.PrivateKey
 	}
 	return cert, key, nil
 }
+
+// Generate public key from keyPEM string
+func ComputePublicKey(certPEM string) (string, error){
+	cert, err := ParseCert(certPEM)
+	if err != nil {
+		return "", err
+	}
+	pubKey := cert.PublicKey.(interface {})
```

Comment:
interface{} means that is a blank interface, absolutely anything will match this, you dont need a typecast, just pass whatever cert.PublicKey is and it will work.